merlin/config/utils.py

```diff
@@ -53,26 +54,31 @@ def is_redis_broker(broker: str) -> bool:
     return broker in ["redis", "rediss", "redis+socket"]
 
 
+def determine_priority_map(broker_name: str) -> Dict[Priority, int]:
+    """
+    Returns the priority mapping for the given broker name.
+
+    :param broker_name: The name of the broker that we need the priority map for
+    :returns: The priority map associated with `broker_name`
+    """
+    if is_rabbit_broker(broker_name):
+        return {Priority.LOW: 1, Priority.HIGH: 9, Priority.RETRY: 10}
```

Comment:
Do you want to add a MID kw here, since it is one of the options?

Reply:
I probably should instead of having it just default to Priority of 5. It would be easier to see what's going on this way.